elb_log_ingestor/stats.py

```diff
@@ -0,0 +1,120 @@
+"""
```

Comment:
It would be good to add some doc strings about each stat so it's easy for humans.